src/node/blockmanager_args.cpp

```diff
@@ -5,8 +5,15 @@
 #include <node/blockmanager_args.h>
 
 #include <common/args.h>
+#include <node/blockstorage.h>
+#include <tinyformat.h>
+#include <util/translation.h>
 #include <validation.h>
 
+#include <cstdint>
+#include <memory>
```

Comment:
Seems like it's a bug: https://github.com/include-what-you-use/include-what-you-use/issues/648

Since it sounds like it's going to be fixed in an upcoming version, I'll remove the `memory` include.

Reply:
Huh, yeah, seems like it. But I guess the issue is similar to what is happening here?